packages/http-client-csharp/generator/.editorconfig

```diff
@@ -0,0 +1,204 @@
+# editorconfig.org
```

Comment:
That's another one I think it would be good to see if we could standarize with the visual studiop extension(I think you have more things enabled here too so we can see if it applies to vs too

Reply:
Given we are using very different dotnet frameworks it might make sense to remain separate but will leave that up to the owners of the vs extension.

Reply:
yeah that's fine for now we can see  later if we want